### ISSUE TITLE
GestureHandler already initialized for root view

### ISSUE THREAD

**Issue by zinoviev** [kmagiera/react-native-gesture-handler#172]
Hi there, 

I'm basically trying to replicate native driver example from docs:

`const PanAndRotate = () => (
  <PanGestureHandler onGestureEvent={Animated.event({ ... }, { useNativeDriver: true })}>
    <Animated.View>
      <RotationGestureHandler onGestureEvent={Animated.event({ ... }, { useNativeDriver: true })}>
        <Animated.View style={animatedStyles}/>
      </RotationGestureHandler>
    </Animated.View>
  </PanGestureHandler>
);`

and get this error. Maybe I'm missing something, is there any working samples with native driver and Animated.Views? Without native driver seems it's working for me, but I wish to get more animation performance.

**Comment by osdnk** [comment]
Would you mind give us a wider context of this issue? Where do insert `PanAndRotate` component? 
I have that admit that this issue provided me to notice a bug in Example App (https://github.com/kmagiera/react-native-gesture-handler/pull/176)

If you want to see how it should work, you may checkout on this branch (or wait, it gonna be merged soon) and see ["Scale, rotate & tilt"](https://github.com/kmagiera/react-native-gesture-handler/blob/use-native-driver-fix/Example/scaleAndRotate/index.js)
Thank you for this issue.
Please let me know  if you still face the problem

**Comment by osdnk** [comment]
Yes, thank you for you suggestion. I think you're right. Link to example I mentioned above is related to already fixed example.

**Comment by osdnk** [comment]
@zinoviev,
Thank you :)
